airflow/cli/commands/celery_command.py

```diff
@@ -96,6 +99,31 @@ def _serve_logs(skip_serve_logs: bool = False):
         sub_proc.terminate()
 
 
+@after_setup_logger.connect()
+def logger_setup_handler(logger, **kwargs):
+    # Setup levels at which logs go to stderr and stdout if required
+    if conf.has_option("logging", "reset_and_split_logging") and conf.get(
+        "logging", "reset_and_split_logging"
+    ):
+        airflow_formatter = logging.Formatter(conf.get("logging", "log_format"))
```

Comment:
Which formatter is actually used by Celery at the moment?
I think they have some different formatter by default, but if we are making this change probably it is good idea to format Celery logs with same formatter.

then you can do settings.LOG_FORMAT here?

Reply:
I think it is this one:
```"""[%(asctime)s: %(levelname)s/%(processName)s] %(task_name)s[%(task_id)s]: %(message)s"""```
From:
https://github.com/celery/celery/blob/025c1d4f9a1208a9d3bb01b1129a0f48234d3df0/celery/app/defaults.py#L17

Reply:
I would use either DEFAULT_TASK_LOG_FMT const or settings.LOG_FORMAT const here

Reply:
Please confirm whether current solution is ok.